record_data: {
  destination: "/home/pracsys/Dropbox/transfer/",
  filename: "training_data/combined_all",
  requested_data_ids: [1,2,3,4,5,7,8,9], # description in record_data.cpp
  data_window_size: 10,
  marker_no: 6,
  save_images: false,
  ref_marker_id: 0,
  obj_marker_id: 5
}

play_bags: {
  bag_directory: "/home/pracsys/Dropbox/transfer",
  bag_files: ["cb_20_01","cb_20_02","cb_20_03","cb_20_04","cb_20_05","cb_20_06","cb_20_07","cb_20_08","cb_20_09","cb_20_10","cb_20_11","cb_20_12","cb_20_13","cb_20_14","cb_20_15","cb_20_16","cb_20_17","cb_20_18","cb_20_19","cb_20_20","cb_20_21","cb_20_22","cb_20_23","cb_20_24","cb_20_25"]}
